### ISSUE TITLE
Flow should open on a new page when clicked

### ISSUE THREAD

**Issue by gsambrotta** [kendraio/kendraio-app#384]
This is more a proposal open to discussion.

Most of the time I find myself in a situation in which I want to check several flows at the same time. Or I want to check out a flow but still consult the list of available flows. 
Right now when I Launch a flow, it will open the flow on the same page and the flow list is not available anymore.
I would prefer to open the flow in a separate tab. 
It might be just my preference as a developer but I if we don't set up "open in a new tab" as default behaviour, we should implement it at least as an option.

**Comment by dahacouk** [comment]
I totally agree that "open in a new tab" should be an option when we click on the Launch button for a Flow. Also, any button should be able to have that option. I don't think this should be default behaviour but definitely an option, yes. Good idea.

In macOS I can press a the command key and click to get the "open in a new tab" really quickly. So, I'd love that to work for any button that takes you to a new Flow/page in Kendraio App.

**Comment by gsambrotta** [comment]
After some research, this is what I found out so far:

- to create link with default browser option to open in a new Tab, an `href` attribute is needed. Here an example of a link with button UI from [Angular Material documentation](https://material.angular.io/components/button/overview) `<a mat-button href="https://www.google.com/" target="_blank">Link</a>`

- Our button style come from Action Block. This block is render as `button` tag in most of the case and as `a` tag if `color=link`. This is irrelevant, tough, as the `a` tag does not use `href` but instead use `onClick`.

- To open link we use the Launch Block, In combination with the Button Block. 

In my opinion, we should rewrite the Action Block and allow choosing the user if is a simple button or if is a link. And if is a link, it should behave as normal link, regardless if is opening another flow or is opening an external link.
We should not use Launch for this. 
This means that the Link should also have the option to set the context of the opened flow, as it does now. 
But this can be achieved simply with URL params. I think we should stay as close as possible to the natural implementation of the browsers.

**Comment by gsambrotta** [comment]
@lukestanley @CodeKrakken I would be happy to hear your opinion about this

**Comment by CodeKrakken** [comment]
@gsambrotta That makes sense to me. Are you suggesting we add an option for a basic link into the existing Launch block, or introduce a new Link block?

**Comment by dahacouk** [comment]
I want to prioritise this issue as it is getting annoying for me. Let's talk about this on Monday 2024-07-01. @lukestanley @gsambrotta @CodeKrakken Cheers!

**Comment by lukestanley** [comment]
Currently working on a branch here: https://github.com/kendraio/kendraio-app/tree/link-action-block - I am fixing an unrelated unit test issue.
